src/components/Layout/Sidebar.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Avatar from './Avatar';
+import { UserProfile } from 'services/user';
+
+import styles from './Layout.module.css';
+
+interface ComponentProps {
```

Comment:
Should we name it `SidebarProps`? 
```suggestion
interface SidebarProps {
```

Reply:
Updated in [95c8bb2](https://github.com/hanam1ni/react-surveyor/pull/60/commits/95c8bb2aab315c475ed8440499f07b711892a192)